### DISCUSSION TITLE
Does the self hosted version log requests on local machine docker installation?

### DISCUSSION THREAD

**Discussion by arjunhadke7** [plausible/analytics#2574] [Self-Hosted Support]
Does the self hosted version log requests on local machine docker installation? Or does it work only on a cloud based Docker installation like on Digitalocean or others?

I was trying the self hosted version while Docker is installed on my local machine and I have added the script and the code snippet to a website hosted on shared hosting. While I make the requests (clean new requests and even new requests from separate browsers/incognito mode), I am still seeing the green dot which says, 'waiting for first pageview on website.com'.

**Comment by ruslandoga** [comment]
@arjunhadke7 👋

Is your local machine accessible on the public internet, does it have a public IP address? If not, your setup with Plausible running on your local machine (in your local network) won't work since the website on shared hosting won't have any way to access it.

**Reply by arjunhadke7** [reply]
Hi @ruslandoga 
Thanks for the quick response. I was just testing it out and figured it out a little later. Thanks a lot for your reply, it clarifies what I had been reading. Great day ahead!
